nahusenayElias/Team_4
# DRD-111: About page styling

## Related ticket
[DRD-111](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-111)

## In this PR: 
- increased body text font size
- added routing for Projects page /projects to display the project container in a single page
- added buttons to direct to Services, Projects and Contact page
- centered text paragraphs content on the page
- added some content in Drupal
- centered the info cards on About page and made them into single column when viewed with a small screen

## Testing instructions
- checkout branch
- lando db-import DRD-111.sql.gz
- Check the about page: test that buttons work, test with different screen sizes

[DRD-111]: https://edu-team4-react24k.atlassian.net/browse/DRD-111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
